### ISSUE TITLE
Publish 1.0.6 to npm

### ISSUE THREAD

**Issue by EugeneDraitsev** [react-stack/redux-storage-engine-reactnativeasyncstorage#3]
Hi, could you please publish 1.0.6 version to npm?

**Comment by mdanka** [comment]
@guns2410 @ptomasroos  Same ask here - could you publish the latest version by any chance?

Thank you!

**Comment by mdanka** [comment]
@EugeneDraitsev FYI in the meantime you can install the latest commit by adding this dependency line to your `package.json`:

```
"redux-storage-engine-reactnativeasyncstorage": "git://github.com/react-stack/redux-storage-engine-reactnativeasyncstorage.git#5acf34b2e14ad59889bfd063cd7e6891d53158ba",
```

**Comment by ptomasroos** [comment]
Published as 1.0.7 https://www.npmjs.com/package/redux-storage-engine-reactnativeasyncstorage

**Comment by mdanka** [comment]
Amazing, thank you!
